src/Game/Network/Services/ChannelService.cs

```diff
@@ -124,8 +127,35 @@ await session.Player.ChatSession.SendAsync(new SChatMessageAckMessage(ChatType.C
         [MessageHandler(typeof(CQuickStartReqMessage))]
         public Task CQuickStartReq(GameSession session, CQuickStartReqMessage message)
         {
-            //ToDo - Logic
-            return session.SendAsync(new SServerResultInfoAckMessage(ServerResult.FailedToRequestTask));
+            if (session.Player.Channel == null)
+                return session.SendAsync(new SServerResultInfoAckMessage(ServerResult.FailedToRequestTask));
+
+            var rooms = session.Player.Channel.RoomManager.Select(r => r.Map<Room, RoomDto>()).ToArray();
```

Comment:
This is not needed and just a waste of allocations. You can use the RoomManager directly.